railgun.yml

```diff
@@ -0,0 +1,18 @@
+# https://dev-accel.shopify.io/dev/railgun/Railgun-Config
+name: gozk-recipes
+
+vm:
+  image:      /opt/dev/misc/railgun-images/default
+  ip_address: 192.168.64.87
+  memory: 2G
```

Comment:
Do we really need 2G memory? (Not to say this really matters)

Reply:
I guess this is legacy. Whatever :)

Reply:
Trying with something smaller.

Reply:
I tried some different values the minimum I could use and have the services start was `200M` and so I'm running with that.